docs/src/snap/howto/cis-hardening.md

```diff
@@ -282,6 +347,3230 @@ the sets, including the dqlite specific checks in the output.
+
+```
+/bin/sh -c 'if test -e /etc/kubernetes/scheduler.conf; then stat -c %U:%G /etc/kubernetes/scheduler.conf; fi'
+```
+
+**Expected output:**
+
+```
+root:root
+```
+
+##### Control 1.1.17
+
+**Description:**
+
+Ensure that the controller-manager.conf file permissions are
+set to 600 (Automated)
+
+
+**Remediation:**
+
+Run the following command on the control plane node.
+
+`chmod 600 /etc/kubernetes/controller.conf`
+
+
+**Audit (as root):**
+
+```
+/bin/sh -c 'if test -e /etc/kubernetes/controller.conf; then stat -c permissions=%a /etc/kubernetes/controller.conf; fi'
+```
+
+**Expected output:**
+
+```
+permissions=600
+```
+
+##### Control 1.1.18
+
+**Description:**
+
+Ensure that the controller-manager.conf file ownership is
+set to root:root (Automated)
+
+
+**Remediation:**
+
+Run the following command on the control plane node.
+
+`chown root:root /etc/kubernetes/controller.conf`
+
+
+**Audit (as root):**
+
+```
+/bin/sh -c 'if test -e /etc/kubernetes/controller.conf; then stat -c %U:%G /etc/kubernetes/controller.conf; fi'
+```
+
+**Expected output:**
+
+```
+root:root
+```
+
+##### Control 1.1.19
+
+**Description:**
+
+Ensure that the Kubernetes PKI directory and file ownership
+is set to root:root (Automated)
+
+
+**Remediation:**
+
+Run the following command on the control plane node.
+
+`chown -R root:root /etc/kubernetes/pki/`
+
+
+**Audit (as root):**
+
+```
+find /etc/kubernetes/pki/ | xargs stat -c %U:%G
+```
+
+**Expected output:**
+
+```
+root:root
+```
+
+##### Control 1.1.20
+
+**Description:**
+
+Ensure that the Kubernetes PKI certificate file permissions
+are set to 600 (Manual)
+
+
+**Remediation:**
+
+Run the following command on the control plane node.
+
+`chmod -R 600 /etc/kubernetes/pki/*.crt`
+
+
+**Audit (as root):**
+
+```
+find /etc/kubernetes/pki/ -name '*.crt' | xargs stat -c permissions=%a
+```
+
+**Expected output:**
+
+```
+permissions=600
+```
+
+##### Control 1.1.21
+
+**Description:**
+
+Ensure that the Kubernetes PKI key file permissions are set
+to 600 (Manual)
+
+
+**Remediation:**
+
+Run the following command on the control plane node.
+
+`chmod -R 600 /etc/kubernetes/pki/*.key`
+
+
+**Audit (as root):**
+
+```
+find /etc/kubernetes/pki/ -name '*.key' | xargs stat -c permissions=%a
+```
+
+**Expected output:**
+
+```
+permissions=600
+```
+
+#### API Server
+
+##### Control 1.2.1
+
+**Description:**
+
+Ensure that the --anonymous-auth argument is set to false
+(Manual)
+
+
+**Remediation:**
+
+Edit the API server configuration file /var/snap/k8s/common/args/kube-apiserver
+on the control plane node and set the following argument.
+
+`--anonymous-auth=false`
+
+
+**Audit (as root):**
+
+```
+/bin/ps -ef | grep kube-apiserver | grep -v grep
+```
+
+**Expected output:**
+
+```
+--anonymous-auth=false
+```
+
+##### Control 1.2.2
+
+**Description:**
+
+Ensure that the --token-auth-file parameter is not set
+(Automated)
+
+
+**Remediation:**
+
+Follow the documentation and configure alternate mechanisms
```

Comment:
make distinction to follow Kubernetes docs

Reply:
See https://github.com/canonical/k8s-snap/pull/802/files#diff-da8b48161c214d6854a37a2ef8a664491413a6b43793ba6e9e6319a49da719ecR975